dags/process_elife_zip_dag.py

```diff
@@ -197,14 +197,28 @@ def wrap_article_in_libero_xml_and_send_to_service(**context) -> None:
         root
     )
 
+    return etree.tostring(xml, xml_declaration=True, encoding='UTF-8')
```

Comment:
I see this is not explicitly logged but I trust it would be accessible in someway in the Airflow UI as it's the return value of a task

Reply:
Correct, task return items are logged also.